app/tests/base.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+import json
+from fastapi.testclient import TestClient
+from mongoengine import connect, disconnect
+from main import app
+
+# client = TestClient(app)
+
+
+class BaseTestCase(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        connect("mongoenginetest", host="mongomock://127.0.0.1:8000")
+        cls.client = TestClient(app)
+
+    @classmethod
+    def tearDownClass(cls):
+        disconnect()
+
+    def setUp(self):
+        data = open("app/dummy_data/homework_quiz.json")
```

Comment:
Also this file is not committed to this branch?

Reply:
This file was added beforehand in the PR where tests were first added: https://github.com/avantifellows/quiz-backend/pull/25/files